Frameworks/EOF/ERRest/Sources/er/rest/ERXRestRequestNode.java

```diff
@@ -276,9 +279,9 @@ else if (_value != null) {
 				for (ERXRestRequestNode child : _children) {
 					String name = child.name();
 					Object value = child.toNSCollection(delegate, associatedObjects);
-					// if (value != null) {
-					dict.put(name, value);
-					// }
+					if (value != NSKeyValueCoding.NullValue || ERXProperties.booleanForKeyWithDefault("ERXRest.includeNullValues", true) == true) {
```

Comment:
_== true_ part is not needed as booleanForKeyWithDefault already returns a boolean ;)